modules/github-bots/sdk/handlers.go

```diff
@@ -44,4 +50,5 @@ const (
 
 	// LoFo events
 	WorkflowRunArtifactEvent EventType = "dev.chainguard.lofo.workflow_run_artifacts"
+	WorkflowRunWithLogsEvent EventType = "dev.chainguard.lofo.workflow_run_logs"
```

Comment:
Should we drop the `..With..` to keep it consistent with the artifact event?
```suggestion
	WorkflowRunLogsEvent EventType = "dev.chainguard.lofo.workflow_run_logs"
```

Reply:
sounds great!